examples/notebooks/beam-ml/run_inference_basic.ipynb

```diff
@@ -1362,6 +1308,408 @@
+        "\n",
+        "import numpy\n",
+        "\n",
+        "from typing import Dict, Text, Any, Tuple, List\n",
+        "\n",
+        "from apache_beam.options.pipeline_options import PipelineOptions\n",
+        "\n",
+        "project = \"<Your Project>\"\n",
+        "bucket = \"<Your Bucket>\"\n",
+        "\n",
+        "save_model_dir_multiply = f'gs://{bucket}/tfx-inference/model/multiply_five/v1/'\n",
+        "\n",
+        "# NOTE: If you get a ContextualVersionConflict error, restart your runtime\n",
+        "# environment (under the menu Runtime->Restart Runtime)\n"
+      ],
+      "metadata": {
+        "id": "zSashIqde3Ar"
+      },
+      "execution_count": 2,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Create and Test a Simple Model\n",
+        "\n",
+        "This creates a model that predicts the 5 times table."
+      ],
+      "metadata": {
+        "id": "ohPytRVDHc94"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Create training data which represents the 5 times multiplication table for 0 to 99. x is the data and y the labels. \n",
+        "x = numpy.arange(0, 100)\n",
+        "y = x * 5\n",
+        "\n",
+        "\n",
+        "# Build a simple linear regression model.\n",
+        "# Note the model has a shape of (1) for its input layer, it will expect a single int64 value.\n",
+        "\n",
+        "input_layer = keras.layers.Input(shape=(1), dtype=tf.float32, name='x')\n",
+        "output_layer= keras.layers.Dense(1)(input_layer)\n",
+        "\n",
+        "model = keras.Model(input_layer, output_layer)\n",
+        "model.compile(optimizer=tf.optimizers.Adam(), loss='mean_absolute_error')\n",
+        "\n",
+        "# Test the model\n",
+        "model.fit(x, y, epochs=4000, verbose=0)\n",
+        "value_to_predict = numpy.array([20, 40, 60, 90], dtype=numpy.float32)\n",
+        "model.predict(value_to_predict)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "RmzuFB1ymqdP",
+        "outputId": "bec9a1d1-3d90-4b5f-9819-d20a7644adce"
+      },
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "1/1 [==============================] - 0s 56ms/step\n"
+          ]
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "array([[100.00245],\n",
+              "       [200.0051 ],\n",
+              "       [300.00775],\n",
+              "       [450.01172]], dtype=float32)"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 7
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Populate the Data into a TensorFlow Proto\n",
```

Comment:
I would cut this section and the `Fit the Model` section - right now this TensorFlow section spends 4 sections on training a TF model and only one on actually using RunInference, I think that's probably backwards from what we want. I'd assume people know how to train a model (or point them to docs that can teach them)